test/integration/raw_datastore_test_impl.dart

```diff
@@ -519,7 +519,7 @@ runTests(Datastore datastore, String namespace) {
         expect(testEmptyCommit(
                namedEntities20Keys, transactional: true, xg: true),
                throwsA(isApplicationError));
-      }, skip: 'Existing failure');
```

Comment:
Are you sure this one is passing? I think the XG transactions have increased their limit of entities one can touch, that's why this negative failure was failing (i.e. it was passing).

Reply:
You are correct -- I thought I had tried this one but running it again shows a failure.  Fixed.